lib/assert.js

```diff
@@ -150,6 +152,14 @@ x.ifError = (err, msg) => {
 	test(!err, create(err, 'Error', '!==', msg, x.ifError));
 };
 
+x.jsxEqual = (val, expected, msg) => {
+	test(deepEqual(val, expected), create(val, expected, '===', msg, x.jsxEqual));
+};
+
+x.notJsxEqual = (val, expected, msg) => {
+	test(!deepEqual(val, expected), create(val, expected, '!==', msg, x.notJsxEqual));
+};
```

Comment:
When looking into documenting these, I realized they're exactly the same as `t.deepEqual()`. So what is the point of these?